### ISSUE TITLE
HipHop VM's php in PATH

### ISSUE THREAD

**Issue by andyli** [travis-ci/travis-ci#6451]
We found that the php in PATH has become HipHop VM in our build:
https://travis-ci.org/HaxeFoundation/haxe/jobs/151090373

```
test php
Changing directory to /home/travis/build/HaxeFoundation/haxe/tests/unit/
php has already been installed.
Command: php [-v]
HipHop VM 3.14.4 (rel)
Compiler: tags/HHVM-3.14.4-0-gcee2aedf82cffa3f4959adf9e0409c9ec7b314cb
Repo schema: c6d3b7c365d7be3fe41c697542afe0a1b1802987
Command exited with 0 in 0s: php [-v]
```

Previously there is no php in PATH.

I guess it makes more sense to have an official php (preferably PHP 5.5 which is installable to trusty using apt-get) in PATH instead of HipHop VM.

**Comment by truell20** [comment]
+1

**Comment by BanzaiMan** [comment]
OK, so it appears that the current image was provisioned in such a state that `phpenv global hhvm` was the last of PHP runtimes to be set up.

Until we fix the image, please run `phpenv global 5.6` or whatever you think is appropriate.

Thanks.
